### ISSUE TITLE
llvm 9.0.0 - release pages on github?

### ISSUE THREAD

**Issue by rubyFeedback** [llvm/llvm-project#38]
Hey folks,

Not sure if this was an oversight or deliberate but on:

https://github.com/llvm/llvm-project/releases/

There is nothing about llvm 9.0.0, only 8.0.1 and so forth.

Particularly what I liked was that we could download 8.0.1 and also
clang and the rt-stuff. This was convenient for me because I
did not have to search the official homepage before I find it :)
(it takes me so much longer to navigate the official homepage
compared to the more generic setup on github). Anyway just
posting this in case it was just forgotten.

**Comment by tstellar** [comment]
These were test releases we did while evaluating the GitHub infrastructure, which is why there are only a few available.  Going forward, all release artifacts will be uploaded to GitHub.

**Comment by jyknight** [comment]
Should we upload the archives for all the past releases too?

**Comment by rubyFeedback** [comment]
I am not sure if/what has changed, but when I look at:

https://github.com/llvm/llvm-project/releases

I can now see entries such as for clang, compiler-rt etc... for llvm 9.0.1.

I am aware that I filed this for 9.0.0 but I believe that this is now obsolete,
and can be closed (if nobody minds; since I started it, I think it may be
fair to close it so that nobody else has to close it).

To the question as for uploading archives for past releases:

- I believe if it is not too much work, then this might be good BUT, at the
least for me own personal use cases, I really only follow the latest stable
releases. So if I have the option between 9.0.1 and 9.0.0 then I will
go with llvm 9.0.1 (I am not really a developer, more a hobbyist, but
I compile everything from source through ruby, with some help from
the Linux from Scratch project).

I do not know whether this helps the LFS project or not (I am not 
an active maintainer) but their current page for llvm is here:

http://www.linuxfromscratch.org/blfs/view/cvs/general/llvm.html

For 8.0.1

I assume they may soon update to llvm 9.0.1 (or perhaps 9.0.0
but I would assume they will move to 9.0.1 anyway).

So to answer jyknight's question: I believe if anyone has the resources
to do so (e. g. time), then adding add-ons for past releases would be
nice BUT personally I do not think it is really necessary for most people.

Big distributions probably maintain their own branches anyway. If I
am wrong then they could of course comment. :)

I will close this now - thanks for the update to 9.0.1 in general! Will
give it a try tomorrow (or coming weekend).
